Readme.md

```diff
@@ -74,17 +75,27 @@ If you intend to do something more advanced with Node.js dependencies, you may w
      }
    }
    ```
+
 2. In your _.csproj_ file, find an `<ItemGroup>` and add `<Vendor Include="node_modules/{package_name}" />` items for each.
    - If the distribution files are located in a _dist_ subdirectory of the package (e.g. _./node_modules/chart.js/dist_), then you are good to go.
-   - Otherwise, include a child element `<Subdirectory>some/other/path</Subdirectory>` 
+   - Otherwise, include a child element `<Subdirectory>some/other/path</Subdirectory>`
+
      ```xml
-         <Vendor Include="node_modules/chart.js">
-           <Subdirectory>dist</Subdirectory>
-         </Vendor>
+     <Vendor Include="node_modules/chart.js">
+       <Subdirectory>dist</Subdirectory>
+     </Vendor>
      ```
+
+     <!-- textlint-disable doubled-spaces -->
+     > [!TIP]
+     > You can include multiple `<Subdirectory>` elements. In this case all of them must exist and they will be all copied into the same output directory.
```

Comment:
This isn't rendering as a tip, but as a blockquote: https://github.com/Lombiq/NPM-Targets/blob/636c3e8bd583392bcc0622bf0ea84b857b90579e/Readme.md#copying-vendor-libraries-into-wwwroot

Reply:
Guess you can't have indented tips.